### PR TITLE
`db seed` GA

### PR DESCRIPTION
**Pull request by molebox**
This PR updates the `db seed` docs to reflect that it is no longer in preview. 

Fixes: #2121 
Closes: #2055

### CODE REVIEW

File: content/300-guides/050-database/300-seed-database.mdx [hunk 1]
```diff
@@ -5,103 +5,65 @@ metaDescription: Learn how to seed your database using Prisma's integrated seedi
...
-
-      ```ts
-      export async function seed() {
-        // Prisma create query to seed models in database
-      }
-      ```
-
-      </tab>
-      <tab>
+Prisma's integrated seeding functionality expects a command in the `"prisma"` key of your `package.json` file. This points to a file which holds your seed script. The location of the file is optional, although we recommend placing it in the `/prisma` folder by convention.
```
Comment (Jolg42) [inline content/300-guides/050-database/300-seed-database.mdx]: Actually 
```
"prisma": {
  "seed": "ts-node prisma/seed.ts"
},
```

So maybe, not sure if it's the best though

```suggestion
Prisma's integrated seeding functionality expects a command in the `"seed"` key in the `"prisma"` key of your `package.json` file. This points to a file which holds your seed script. The location of the file is optional, although we recommend placing it in the `/prisma` folder by convention.
```

File: content/300-guides/050-database/300-seed-database.mdx [hunk 2]
```diff
@@ -185,18 +150,24 @@ To seed your database with TypeScript:
...
 1. Create a new file named `seed.js` in the same location as your `schema.prisma`.
-2. In the `seed.js` file, import Prisma Client, initialize it and create some records. The following example uses a script, but you can also [use an `export`](#requirements-for-seeding-with-typescript-or-javascript):
+2. In the `seed.js` file, import Prisma Client, initialize it and create some records.
+
+  <details><summary>Expand to view example seed script</summary> 
 
+  
    ```js
-   const { PrismaClient } = require('@prisma/client')
+   import { PrismaClient } from '@prisma/client'
```
Comment (Jolg42) [inline content/300-guides/050-database/300-seed-database.mdx]: If its a regular JS project then
```
const { PrismaClient } = require('@prisma/client')
```
only will work by default

Comment (molebox) [reply content/300-guides/050-database/300-seed-database.mdx]: Hmm i guess it depends how the project is setup. If the user has `"types": "module"` in their `package.json` then import syntax is valid. Maybe im just being pedantic about this though and should change it to require instead

Comment (molebox) [reply content/300-guides/050-database/300-seed-database.mdx]: I changed it back to require.

Comment (Jolg42) [reply content/300-guides/050-database/300-seed-database.mdx]: Yes it definitely depends on the project config

File: content/200-concepts/100-components/250-preview-features/080-cli-preview-features.mdx [hunk 3]
```diff
@@ -5,18 +5,12 @@ metaDescription: Prisma CLI features that are currently in preview.
...
 
-## Enabling a Prisma CLI preview feature
+There are currently no Preview CLI features.
 
-To use a Prisma CLI Preview feature, append the `--preview-feature` flag each time you run the command:
+</Admonition>
 
-```terminal
-npx prisma db seed --preview-feature
-```
```
Comment (janpio) [inline content/200-concepts/100-components/250-preview-features/080-cli-preview-features.mdx]: Maybe keep this in, at least commented out, for when we add a CLI preview feature flag again.

File: content/300-guides/050-database/300-seed-database.mdx [hunk 4]
```diff
@@ -5,103 +5,65 @@ metaDescription: Learn how to seed your database using Prisma's integrated seedi
...
 
-## Seeding your database with TypeScript
+<TabbedContent tabs={[<FileWithIcon text="TypeScript" icon="file"/>, <FileWithIcon text="JavaScript" icon="file"/>]}> 
 
-To seed your database with TypeScript:
+ 
+<tab> 
 
+ 
 1. Create a new file named `seed.ts` in the same location as your `schema.prisma`.
```
Comment (janpio) [inline content/300-guides/050-database/300-seed-database.mdx]: The location of this file is not define, the user can put it anywhere they want.

File: content/300-guides/050-database/300-seed-database.mdx [hunk 5]
```diff
@@ -165,14 +130,14 @@ To seed your database with TypeScript:
 
 <a id="compiler-options"></a>
 
-4. Optionally, add additional `--compiler-options` to the script that Prisma uses to run the seed script ([2.18.0](https://github.com/prisma/prisma/releases/tag/2.18.0) and later). This makes `prisma db seed` work with tools that have specific requirements when used with TypeScript, such as Next.js. The name of the script is `ts-node`:
+4. Add the `prisma.seed` to your `package.json` file:
```
Comment (janpio) [inline content/300-guides/050-database/300-seed-database.mdx]:
```suggestion
4. Add the `prisma.seed` key to your `package.json` file:
```
Is `key` the right name for this? Or `property`?

File: content/300-guides/050-database/300-seed-database.mdx [hunk 6]
```diff
@@ -165,14 +130,14 @@ To seed your database with TypeScript:
...
 
    ```js highlight=5;normal
    {
      "name": "my-project",
      "version": "1.0.0",
-     "scripts": {
-       "ts-node": "ts-node --compiler-options \"{\\\"module\\\":\\\"commonjs\\\"}\""
+     "prisma": {
+       "seed": "ts-node --compiler-options \"{\\\"module\\\":\\\"commonjs\\\"}\" prisma/seed.ts"
```
Comment (janpio) [inline content/300-guides/050-database/300-seed-database.mdx]: Are these weird compiler-options still required?

File: content/300-guides/050-database/300-seed-database.mdx [hunk 7]
```diff
@@ -5,103 +5,65 @@ metaDescription: Learn how to seed your database using Prisma's integrated seedi
...
-  allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
-  allowfullscreen
-></iframe>
-
-</div>
+- Provide basic data for validating and using your application in a development environment. This is particularly useful if you are using Prisma Migrate, which sometimes requires resetting your development database.
 
 </TopBlock>
 
 ## Requirements for seeding with TypeScript or JavaScript
```
Comment (janpio) [inline content/300-guides/050-database/300-seed-database.mdx]: The paragraph below is also true if you use a bash script - this is fundamentally how you tell Prisma how to execute your seed script now.

So maybe something like that:

```suggestion
## Configuring your seed script
```

File: content/300-guides/050-database/300-seed-database.mdx [hunk 8]
```diff
@@ -283,9 +290,16 @@ set -ex
...
+ 
+</tab> 
+
+ 
+</TabbedContent>
+
+
 ## Integrated seeding with Prisma Migrate
 
 Prisma Migrate integrates seamlessly with your seeds, assuming you follow the steps in the previous section. When Prisma Migrate resets the development database, seeding is triggered automatically. Prisma Migrate resets the database and triggers seeding in the following scenarios:
```
Comment (janpio) [inline content/300-guides/050-database/300-seed-database.mdx]:
```suggestion
Prisma Migrate integrates seamlessly with your seeds, assuming you follow the steps in the previous section. When Prisma Migrate resets the development database, seeding is triggered automatically. 

Prisma Migrate resets the database and triggers seeding in the following scenarios:
```

File: content/300-guides/050-database/300-seed-database.mdx (hunk 5 above)
Comment (molebox) [reply content/300-guides/050-database/300-seed-database.mdx]: Perhaps property. prisma is the key on that property. But its also another object which has seed as a key 😁

Comment (molebox) [reply content/300-guides/050-database/300-seed-database.mdx]: Maybe field is better actually

File: content/300-guides/050-database/300-seed-database.mdx (hunk 6 above)
Comment (molebox) [reply content/300-guides/050-database/300-seed-database.mdx]: I do not know

**Comment by tomhoule** [comment]
It's hard for me to see through the markup diff, but have the content changes from https://github.com/prisma/docs/pull/2055 been taken into account? I think this is the most important information we need to reflect the new `db seed`.

File: content/300-guides/050-database/300-seed-database.mdx [hunk 9]
```diff
@@ -5,103 +5,65 @@ metaDescription: Learn how to seed your database using Prisma's integrated seedi
...
-## Requirements for seeding with TypeScript or JavaScript
-
-Prisma's integrated seeding functionality expects the following:
-
-<!-- prettier-ignore-start -->
+## Configuring your seed script
 
-1. A `seed.ts` or `seed.js` file in the [same location as your `schema.prisma` file](../../concepts/components/prisma-schema#prisma-schema-file-location).
-2. **One** of the following inside the `seed.js`/`seed.ts` file:
+Prisma's integrated seeding functionality expects a command in the `"seed"` key in the `"prisma"` key of your `package.json` file. This points to a file which holds your seed script. The location of the file is optional, although we recommend placing it in the `/prisma` folder by convention.
```
Comment (tomhoule) [inline content/300-guides/050-database/300-seed-database.mdx]: The `seed` key is any command, there is not necessarily a file in the project involved. I think we could use the content from https://github.com/prisma/docs/pull/2055 as a starting point.

Comment (molebox) [reply content/300-guides/050-database/300-seed-database.mdx]: Hey @tomhoule , this is based off of that PR. Perhaps I wasn't clear in the above. I will re-write it.

**Comment by molebox** [comment]
> It's hard for me to see through the markup diff, but have the content changes from #2055 been taken into account? I think this is the most important information we need to reflect the new `db seed`.

Hey @tomhoule, this whole PR is based off of the work done in #2055. You can check the [preview link](https://deploy-preview-2161--prisma2-docs.netlify.app/docs/guides/database/seed-database) to get a better overview of the changes and how it reads

**Comment by tomhoule** [comment]
Ok thanks, I'll do a review based on the preview link :)

File: content/300-guides/050-database/300-seed-database.mdx [hunk 10]
```diff
@@ -5,103 +5,65 @@ metaDescription: Learn how to seed your database using Prisma's integrated seedi
...
+- Provide basic data for validating and using your application in a development environment. This is particularly useful if you are using Prisma Migrate, which sometimes requires resetting your development database.
 
 </TopBlock>
 
-## Requirements for seeding with TypeScript or JavaScript
-
-Prisma's integrated seeding functionality expects the following:
-
-<!-- prettier-ignore-start -->
+## Configuring your seed script
```
Comment (janpio) [inline content/300-guides/050-database/300-seed-database.mdx]: Looking at the preview, this still feels to specific. This is more about "enabling seeing in Prisma" or "Configuring the seeding command for Prisma to use" or similar.

File: content/300-guides/050-database/300-seed-database.mdx [hunk 11]
```diff
@@ -5,103 +5,65 @@ metaDescription: Learn how to seed your database using Prisma's integrated seedi
...
+</TabbedContent>
 
-      module.exports = seed
-      ```
 
-      </tab>
-      </TabbedContent>
 
-<!-- prettier-ignore-end -->
+## Seeding your database with TypeScript or JavaScript
```
Comment (janpio) [inline content/300-guides/050-database/300-seed-database.mdx]:
```suggestion
## Example seed scripts

Here we suggest some specific seed scripts for different situations. You are free to customize these in any way, but can also use them as presented here:

### Seeding your database with TypeScript or JavaScript
```

File: content/300-guides/050-database/300-seed-database.mdx [hunk 12]
```diff
@@ -251,16 +230,45 @@ To seed your database with JavaScript:
...
+   ```
+
+ 
+</tab> 
+
+ 
+</TabbedContent>
+
 
 ## Seeding your database with a Bash script
```
Comment (janpio) [inline content/300-guides/050-database/300-seed-database.mdx]: Related to https://github.com/prisma/docs/pull/2161/files#r692118122

```suggestion
### Seeding your database via any language (with a Bash script)
```

File: content/300-guides/050-database/300-seed-database.mdx [hunk 13]
```diff
@@ -283,9 +298,18 @@ set -ex
...
+</tab> 
+
+ 
+</TabbedContent>
+
+
 ## Integrated seeding with Prisma Migrate
 
-Prisma Migrate integrates seamlessly with your seeds, assuming you follow the steps in the previous section. When Prisma Migrate resets the development database, seeding is triggered automatically. Prisma Migrate resets the database and triggers seeding in the following scenarios:
+Prisma Migrate integrates seamlessly with your seeds, assuming you follow the steps in the previous section. When Prisma Migrate resets the development database, seeding is triggered automatically. When Prisma Migrate resets the development database, seeding is triggered automatically.
```
Comment (tomhoule) [inline content/300-guides/050-database/300-seed-database.mdx]: Nitpick: the last sentence is repeated here

File: content/300-guides/050-database/300-seed-database.mdx [hunk 14]
```diff
@@ -283,9 +298,18 @@ set -ex
...
 ## Integrated seeding with Prisma Migrate
 
-Prisma Migrate integrates seamlessly with your seeds, assuming you follow the steps in the previous section. When Prisma Migrate resets the development database, seeding is triggered automatically. Prisma Migrate resets the database and triggers seeding in the following scenarios:
+Prisma Migrate integrates seamlessly with your seeds, assuming you follow the steps in the previous section. When Prisma Migrate resets the development database, seeding is triggered automatically. When Prisma Migrate resets the development database, seeding is triggered automatically. 
+
+Prisma Migrate resets the database and triggers seeding in the following scenarios:
 
 - You manually run the `prisma migrate reset` CLI command.
-- The database is reset interactively in the context of using `prisma migrate dev` - for example, as a result of migration history conflicts or database drift.
+- The database is reset interactively in the context of using `prisma migrate dev` - for example, as a result of migration history conflicts or database schema drift.
```
Comment (tomhoule) [inline content/300-guides/050-database/300-seed-database.mdx]: For both `reset` and `dev`, there is a `--skip-seed` CLI flag to skip seeding, I think it's worth quickly mentioning here. Maybe something like:

> When you want to use `prisma migrate dev` or `prisma migrate reset` without seeding, you can pass the `--skip-seed` flag.

**Review by tomhoule — COMMENTED** [review]
It was a lot easier to read with the preview link, and I think it mostly looks good now. I will re-read later and see if we can expand a bit on the introduction, I think it's important to give the right high-level picture: `prisma.seed` key in `package.json` that can be _any_ command, we are not tied to seed scripts.

**Comment by tomhoule** [comment]
Another thing I just noticed is that the document only mentions `db seed` once in the context of an example. Maybe we could present this along the integrated seeding in `Integrated seeding with Prisma Migrate` as all the cases where the seed feature exists. I could draft something.

File: content/300-guides/050-database/300-seed-database.mdx (hunk 13 above)
Comment (molebox) [reply content/300-guides/050-database/300-seed-database.mdx]: Thats not a nitpick! You're totally right to point that out! 🙌

File: content/300-guides/050-database/300-seed-database.mdx (hunk 10 above)
Comment (molebox) [reply content/300-guides/050-database/300-seed-database.mdx]: SEO title FTW: How to seed your database in Prisma

**Comment by molebox** [comment]
> Another thing I just noticed is that the document only mentions `db seed` once in the context of an example. Maybe we could present this along the integrated seeding in `Integrated seeding with Prisma Migrate` as all the cases where the seed feature exists. I could draft something.

That would be wonderful. Alternatively, if you dont have time to do a draft, you could jot some notes down in a comment here on the issue and i could put it together and ask you some questions when you're free

**Review by tomhoule — COMMENTED** [review]
Hi @molebox , sorry it took so long. I think that with these two suggestions, the seed docs will be in a good shape and we can merge.

**Pull request merged**